src/test/TestStringName.cpp

```diff
@@ -0,0 +1,184 @@
+
```

Comment:
// Copyright © 2008-2022 Pioneer Developers. See AUTHORS.txt for details
// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt